src/components/body/sidebar/brands/index.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import { Checkbox } from "@/components/common/checkbox";
+import { useAppDispatch, useAppSelector } from "@/redux/hooks";
+import { setBrands, setFilteredBrands } from "@/redux/slices/brands-slices";
+
+import type { BrandsProps } from "./brands.type";
+
+const Brands: React.FC<{}> = () => {
+  const [searchValue, setSearchValue] = useState<string>("");
```

Comment:
Chỗ này e viết 1 cái hook useDebound để thực hiện delay search nha
https://dev.to/codebayu/react-custom-hooks-usedebounce-4il9

----------------------------------
E mô tả luôn nó hoạt động vào comment trong pull

Reply:
done